index.html

```diff
@@ -232,5 +232,47 @@ <h1 class="s">Skills</h1>
         </li>
       </div>
     </ul>
+    <!-- Footer Form Section -->
+
+    <section class="content-4 center">
+      <h1 class="clr-blue">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h1>
+    </section>
+    <section class="center">
+      <form action="https://formspree.io/f/xeqwrjjn" method="POST">
+        <input
+          type="text"
+          class="in"
+          maxlength="30"
+          name="name"
+          placeholder="&nbsp;&nbsp;&nbsp;Full name"
+          required
+        />
+        <input
+          type="email"
+          class="in"
+          name="email"
+          placeholder="&nbsp;&nbsp;&nbsp;Email address"
+          required
+        />
+        <textarea
+          name="text"
+          id=""
+          cols="30"
+          rows="10"
+          maxlength="500"
+          required
+          placeholder="
+        Hey I’d like to connect to..."
+        ></textarea>
+        <img class="foot" src="imgs/footer.png" alt="illustration" />
+        <br />
+        <button class="btn btn-2 btn-3" type="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] Kindly note that the default type for a button is `submit`, so you can do away that extra code 🙃.